aurora/core.py

```diff
@@ -817,59 +1638,155 @@ def run_aurora(
+                self.vol_pump, # volume of the pump neutrals reservoirs[cm^3]
+                self.L_leak, # leakage conductance from pump neutrals reservoir towards plasma [cm^3/s]
+                self.surf_mainwall_eff, # effective main wall surface area [cm^2]
+                self.surf_divwall_eff, # effective divertor wall surface area [cm^2]
+                self.advanced_PWI_flag, # logic key for PWI model
+                self.Z_main_wall, # atomic number of the main wall material
+                self.Z_div_wall, # atomic number of the divertor wall material
+                self.rn_main_wall, # reflection coefficients for the simulated impurity at the main wall on the time grid
+                self.rn_div_wall, # reflection coefficients for the simulated impurity at the divertor wall on the time grid
+                self.fluxes_main_wall_background, # fluxes for each background species onto the main wall on the time grid [s^-1]
+                self.fluxes_div_wall_background, # fluxes for each background species onto the divertor wall on the time grid [s^-1]
+                self.y_main_wall, # sputtering yields from simulated impurity + background species from the main wall on the time grid
+                self.y_div_wall, # sputtering yields from simulated impurity + background species from the divertor wall on the time grid
+                self.implantation_depth_main_wall, # considered impurity implantation depth in the main wall [A]
+                self.implantation_depth_div_wall, # considered impurity implantation depth in the divertor wall [A]
+                self.n_main_wall_sat, # considered saturation value of the impurity implantation density into the main wall [m^-2]
+                self.n_div_wall_sat, # considered saturation value of the impurity implantation density into the divertor wall [m^-2]
                 self.rvol_lcfs,
                 self.bound_sep,
                 self.lim_sep,
                 self.prox_param,
                 rn_t0=nz_init,  # if omitted, internally set to 0's
+                ndiv_t0=ndiv_init,  # if omitted, internally set to 0       
+                npump_t0=npump_init,  # if omitted, internally set to 0     
+                nmainwall_t0=nmainwall_init,  # if omitted, internally set to 0     
+                ndivwall_t0=ndivwall_init,  # if omitted, internally set to 0     
                 alg_opt=alg_opt,
                 evolneut=evolneut,
                 src_div=self.src_div,
             )
 
         if plot:
+            
+            if radial_coordinate == 'rho_vol':
+                x = self.rvol_grid
+                xlabel = r"$r_V$ [cm]"
+                x_line=self.rvol_lcfs
+            elif radial_coordinate == 'rho_pol':
+                x = self.rhop_grid
+                xlabel=r'$\rho_p$'
+                x_line = 1
 
             # plot charge state density distributions over radius and time
             plot_tools.slider_plot(
```

Comment:
I agree that "higher-level functionality" is expected to go into ImpRad (which however requires one to get into the OMFIT world; not desirable for @antonellozito ). However, we've always wanted to include basic plotting in Aurora! 

BUT I would suggest that we keep most plotting separate from the call to `run_aurora`. Right now, the number of arguments seems unnecessarily large. I'll suggest some changes in a commit.

Reply:
@antonellozito could you please specify in the `run_aurora docsstring what are the input options that need to be turned on in order to have each of the outputs? (e.g. `rcld_sput_rate` is produced only when PWI model is turned on, etc.)

Reply:
@fsciortino @odstrcilt my aim was to provide an option which, optionally, automatically produces plots when calling the simulation. I stress that this is optional (i.e. plots are produced only when explictly doing out = asim.run_aurora(D_z, v_z,plot = True), not when doing out = asim.run_aurora(D_z, v_z)), and I though it might be useful to have something which automatically does it, instead of manually calling e.g. the function for doing a slider plot later. Of course users can still manually produce plots as they wish.
And also, I agree with @fsciortino in keeping Aurora itself and the OMFIT functionalities just separated.